cloud/services/subnets/client.go

```diff
@@ -54,7 +54,7 @@ func newSubnetsClient(settings azure.ClientSettings, authorizer autorest.Authori
 
 // Get gets the specified subnet by virtual network and resource group.
 func (ac *AzureClient) Get(ctx context.Context, resourceGroupName, vnetName, snName string) (network.Subnet, error) {
-	return ac.subnets.Get(ctx, resourceGroupName, vnetName, snName, "")
+	return ac.subnets.Get(ctx, resourceGroupName, vnetName, snName, "networkSecurityGroup,routeTable")
```

Comment:
what does this do? and why do we need it given that we're also doing a Get on the nsg and rt separately?

Reply:
That will fetch the full object structure for those two sub resources (excluding tags). This is super useful for retrieving name and using that to call a GET with name and resource group.